backend/services/allocationService.js

```diff
@@ -266,6 +269,44 @@ const serviceFactory = (elite2Api, keyworkerApi, offenderSearchResultMax) => {
     }
   }
 
+  const searchOffendersPaginated = async (context, { agencyId, keywords, locationPrefix, pageRequest }) => {
+    const availableKeyworkers = await keyworkerApi.availableKeyworkers(context, agencyId)
+    log.debug('Response from available keyworker request')
```

Comment:
As below - we ought to not have debug statements unless they add something useful.

I know there are some dotted around - we ought to get rid of them unless we think they are useful, but that can be done some other time.

Reply:
sure will remove, was copied from the original method.  Will remove from those too